contrib/selinux/syslog_ng.sh

```diff
@@ -23,9 +23,15 @@
 # 243: invalid plugin encountered #1
 # 242: invalid plugin encountered #2
 # 241: invalid plugin encountered #3
+# 240: include file cannot be loaded
 
 cd $( dirname $( readlink -f "${0}" ) )
-[ -f include/common.sh.inc ] && source include/common.sh.inc
+if [ -f include/common.sh.inc ]; then
```

Comment:
With this change "common.sh.inc" became required. (It was optional before.)

What is the ordering behind keeping a function in "syslog_ng.sh" or move it to common?

Reply:
Testing has revealed that there can be cases when the include file can't be sourced, and in that case it's better to fail early in a predictable way.

The parts of code have been moved to the include file so that plugins can make use of it as well.